Misc/NEWS.d/next/Build/2017-09-13-08-52-58.bpo-31429._vO3LJ.rst

```diff
@@ -0,0 +1 @@
+[WIP] Add --with-ssl-default-suites
```

Comment:
Probably needs the WIP removed :-)

Reply:
Oh, I forgot to update the old blurb. I have created a new one and also added 3.7 whatsnew.